FirebaseCore/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ target_compile_definitions(
 target_link_libraries(
   FirebaseCore PRIVATE
   "-framework Foundation"
+  "-framework OSLog"
```

Comment:
We should be able to remove this now that we figured out https://github.com/google/GoogleUtilities/pull/203.